registrasion/forms.py

```diff
@@ -168,7 +168,7 @@ def ProductsForm(category, products, sold_out_products=None, disabled_products=N
     cat = inventory.Category
     RENDER_TYPES = {
         cat.RENDER_TYPE_QUANTITY: _QuantityBoxProductsForm,
-        cat.RENDER_TYPE_RADIO: _RadioButtonProductsForm,
+        cat.RENDER_TYPE_RADIO: get_form_class_from_settings('RADIO_BUTTON_PRODUCTS_FORM', _RadioButtonProductsForm),
```

Comment:
two things:

  - can we prefix these settings with `REGISTRASION_`
  - should we apply the same logic to all render types while we're here?

Reply:
Agree with both!

Reply:
Fixed in:

- 9bb88d5c1c61067295b9753aac182290db87e1d2
- 1fd10a71f04ee3a36013811bbb26321e52e02f02